cfg/ck8s-cis-1.24/node.yaml

```diff
@@ -0,0 +1,476 @@
+          bin_op: or
+          test_items:
+            - flag: "--read-only-port"
+              path: '{.readOnlyPort}'
+              compare:
+                op: eq
+                value: 0
+            - flag: "--read-only-port"
+              path: '{.readOnlyPort}'
+              set: false
+        remediation: |
+          Edit the kubelet configuration file
+          $kubeletconf on each worker node and set the following argument.
+
+          `--read-only-port=0`
+
+          Restart the kubelet service.
+
+          For example, `snap restart k8s.kubelet`
+        scored: false
+
+      - id: 4.2.5
+        text: "Ensure that the --streaming-connection-idle-timeout argument is not set to 0 (Manual)"
+        audit: "/bin/ps -fC $kubeletbin"
+        audit_config: "/bin/cat $kubeletconf"
+        tests:
+          test_items:
+            - flag: --streaming-connection-idle-timeout
+              path: '{.streamingConnectionIdleTimeout}'
+              compare:
+                op: noteq
+                value: 0
+            - flag: --streaming-connection-idle-timeout
+              path: '{.streamingConnectionIdleTimeout}'
+              set: false
+          bin_op: or
+        remediation: |
+          Edit the kubelet configuration file
+          $kubeletconf on each worker node and set the following argument.
+
+          `--streaming-connection-idle-timeout=5m`
+
+          Restart the kubelet service.
+
+          For example, `snap restart k8s.kubelet`
+        scored: false
+
+      - id: 4.2.6
+        text: "Ensure that the --protect-kernel-defaults argument is set to true (Automated)"
+        audit: "/bin/ps -fC $kubeletbin"
+        audit_config: "/bin/cat $kubeletconf"
+        tests:
+          test_items:
+            - flag: --protect-kernel-defaults
+              path: '{.protectKernelDefaults}'
+              compare:
+                op: eq
+                value: true
+        remediation: |
+          Edit the kubelet configuration file
+          $kubeletconf on each worker node and set the following argument.
+
+          `--protect-kernel-defaults=true`
+
+          Restart the kubelet service.
+
+          For example, `snap restart k8s.kubelet`
+        scored: true
+
+      - id: 4.2.7
+        text: "Ensure that the --make-iptables-util-chains argument is set to true (Automated)"
+        audit: "/bin/ps -fC $kubeletbin"
+        audit_config: "/bin/cat $kubeletconf"
+        tests:
+          test_items:
+            - flag: --make-iptables-util-chains
+              path: '{.makeIPTablesUtilChains}'
+              compare:
+                op: eq
+                value: true
+            - flag: --make-iptables-util-chains
+              path: '{.makeIPTablesUtilChains}'
+              set: false
+          bin_op: or
+        remediation: |
+          Edit the kubelet configuration file
+          $kubeletconf on each worker node and
+          remove the --make-iptables-util-chains argument.
```

Comment:
should this really be removed or should this explicitly set to true as the text suggests?